cmd_config.go

```
@@ -0,0 +1,240 @@
+	// Edit config command (opens in default editor)
+	editCmd := &cobra.Command{
+		Use:   "edit",
+		Short: "Edit configuration file",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Check if config exists
+			if _, err := os.Stat(configPath); os.IsNotExist(err) {
+				GlobalLogger.Warning.Printf("Configuration file does not exist: %s\n", configPath)
+				fmt.Print("Create it now? (y/N): ")
+				
+				var response string
+				fmt.Scanln(&response)
```

golangci-lint (1.64.8)

Error return value of `fmt.Scanln` is not checked

(errcheck)